runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -350,7 +350,28 @@ void openGroupConfigPanel(PluginListItem listItem, Config config, ConfigDescript
 					max = range.max();
 				}
 
-				SpinnerModel model = new SpinnerNumberModel(value, min, max, 1);
+				SpinnerModel model;
```

Comment:
This should be simplified to a constraint before setting model rather than a try/catch after .